takwimu/templates/takwimu/profile_page.html

```diff
@@ -12,174 +12,14 @@
 
 
 {% block content %}
-
-  {% include 'takwimu/_includes/profile/hero.html' %}
-
-  {# Loads all the topics #}
-  {% for topic in page.body %}
-      {% include 'takwimu/_includes/profile/topic_detail.html' %}
-  {% endfor %}
-
-  {% include 'takwimu/_includes/profile/sections_list.html' %}
-
-  {% include 'takwimu/_includes/sections/support-services/button.html' %}
-
-  {% include 'takwimu/_includes/sections/stories/index.html' %}
-
-  <!-- Country profile country select -->
-  {% include 'takwimu/_includes/sections/explore/_index.html' with extra_css_class="grayscale" %}
-
-  {% include 'takwimu/_includes/sections/updates.html' %}
-
-  <div id="takwimuViewCountry"></div>
-
+    <div id="takwimuAnalysisPage"></div>
```

Comment:
There is already `{% body_id %}` and `{% body_class %}` blocks that we can use to identify **pages**.

Reply:
I am not sure what your suggesting here.
If your suggesting to use the body_id to render dom, then we may end up overwriting any body javascript unless there is a content id.

Reply:
No, what I meant was if you use `{%body_id%}analysis{%body_id%}` in the template, then the rendered html will have `<body id="analysis">` and then we can use that to know it's analysis page.